File: docs/test-beds/swift/fixture.swift
// Swift test bed
import Foundation

struct Box<T> {
    let value: T
}

class Container<T> {
    private var items: [Box<T>] = []
    func add(_ item: Box<T>) { items.append(item) }
    func size() -> Int { return items.count }
}

class Helper {
    static func log(_ o: Any) { print(o) }
}

func process(_ input: Box<String>) -> Container<String> {
    let c = Container<String>()
    c.add(input)
    Helper.log(c)
    return c
}

let FOO: Int = 42

File: docs/test-beds/python/fixture.py
"""Python test bed."""
from helper import Helper

class Box:
    value: object

class Container:
    items: list
    def add(self, item: Box) -> None:
        self.items.append(item)
    def size(self) -> int:
        return len(self.items)

def process(input: Box) -> Container:
    c = Container()
    c.add(input)
    Helper.log(c)
    return c

FOO: int = 42
